.phive/phars.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <phive xmlns="https://phar.io/phive">
-  <phar name="composer-normalize" version="^2.15.0" installed="2.15.0" location="./.phive/composer-normalize" copy="true"/>
+  <phar name="composer-normalize" version="^2.18.0" installed="2.18.0" location="./.phive/composer-normalize" copy="true"/>
   <phar name="phar-io/phive" version="^0.14.5" installed="0.14.5" location="./.phive/phive" copy="true"/>
-  <phar name="php-cs-fixer" version="^3.2.1" installed="3.2.1" location="./.phive/php-cs-fixer" copy="true"/>
-  <phar name="phpcs" version="^3.6.0" installed="3.6.0" location="./.phive/phpcs" copy="true"/>
-  <phar name="phpmd" version="^2.10.2" installed="2.10.2" location="./.phive/phpmd" copy="true"/>
-  <phar name="psalm" version="^4.10.0" installed="4.10.0" location="./.phive/psalm" copy="true"/>
+  <phar name="php-cs-fixer" version="^3.3.2" installed="3.3.2" location="./.phive/php-cs-fixer" copy="true"/>
+  <phar name="phpcs" version="^3.6.1" installed="3.6.1" location="./.phive/phpcs" copy="true"/>
+  <phar name="phpmd" version="^2.11.0" installed="2.11.0" location="./.phive/phpmd" copy="true"/>
+  <phar name="psalm" version="^4.10.0" installed="4.15.0" location="./.phive/psalm" copy="true"/>
```

Comment:
Should `version` be upped as well as `installed` as it has been for the other tools?